docs/Running_Goerli_Nitro_Node.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: Running_Nitro_Node
```

Comment:
would be good for us to add a redirect from the old page to the new when this gets included to the docs page

Reply:
Good point, added disambiguation page at same link